examples/llama.vim

```diff
@@ -93,6 +93,33 @@ let s:default_config = {
 
 let g:llama_config = get(g:, 'llama_config', s:default_config)
 
+let s:nvim_ghost_text = exists('*nvim_buf_get_mark')
+let s:vim_ghost_text = has('textprop')
+
+if s:vim_ghost_text
+    let s:hint_hlgroup = 'llama_hl_hint'
+    let s:info_hlgroup = 'llama_hl_info'
```

Comment:
This is more inline with the naming convention that we normally use in the project:

```suggestion
    let s:hlgroup_hint = 'llama_hl_hint'
    let s:hlgroup_info = 'llama_hl_info'
```